crates/turbopack-core/src/resolve/options.rs

```diff
@@ -6,7 +6,10 @@ use turbo_tasks::{
     debug::ValueDebugFormat, primitives::StringVc, trace::TraceRawVcs, TryJoinIterExt, Value,
     ValueToString, ValueToStringVc,
 };
-use turbo_tasks_fs::{glob::GlobVc, FileSystemPathVc};
+use turbo_tasks_fs::{
+    glob::{Glob, GlobVc},
+    FileSystemPathVc,
+};
```

Comment:
Rouge edit? Clippy is complaining about unused `Glob`.

```suggestion
use turbo_tasks_fs::{glob::GlobVc, FileSystemPathVc};
```